pkgs/tools/filesystems/file-rename/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchurl, perlPackages, makeWrapper }:
+
+perlPackages.buildPerlPackage rec {
+  name = "File-Rename-0.20";
+
+  src = fetchurl {
+    url = "mirror://cpan/authors/id/R/RM/RMBARKER/${name}.tar.gz";
+    sha256 = "1cf6xx2hiy1xalp35fh8g73j67r0w0g66jpcbc6971x9jbm7bvjy";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  postInstall = ''
+    wrapProgram $out/bin/rename \
+      --prefix PERL5LIB : $out/lib/perl5/site_perl
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Perl extension for renaming multiple files";
+    homepage = http://search.cpan.org/~rmbarker;
+    license = licenses.artistic1;
+    maintainer = with maintainers; [ peterhoeg ];
```

Comment:
`maintainer` isn't a valid meta field, I'm working on a fixup push. This also exposed a bug in ofborg (https://github.com/NixOS/ofborg/issues/73).

Reply:
Arrghh, yes, it's ```maintainers```. Which ```nix-build``` doesn't notice. Any thoughts on how to catch that locally?

Reply:
Thanks for fixing master!

Reply:
You might be able to put `{ checkMeta = true; }` in your Nixpkgs configuration to catch it, otherwise I'm wanting to be able to let people run all the ofborg meta checks locally. That is a bit of a ways out though. In the mean time, I'd recommend just letting ofborg evaluate the PR before merging :) (if there is an ofborg problem, please ping me)